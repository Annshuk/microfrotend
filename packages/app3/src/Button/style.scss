@font-face {
  font-family: 'JeanSunHo';
  src: url('./JeanSunHo.ttf');
  font-weight: 700;
}

.back{
	margin: 0px 15px;
}