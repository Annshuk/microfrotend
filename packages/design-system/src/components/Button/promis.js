const myFakePromise = async () => true;

export { myFakePromise };
