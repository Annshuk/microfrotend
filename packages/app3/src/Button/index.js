import styled from "styled-components";
import "./style.scss";

const Button = ({ className = "" }) => (
    <StyledButton className={ "funny-button " + className }>
        Hello from app1
    </StyledButton>
);
export default Button;

const StyledButton = styled.button`
  font-family: JeanSunHo;
  color: red;
  font-size: 50px;
  height: 80px;
`;