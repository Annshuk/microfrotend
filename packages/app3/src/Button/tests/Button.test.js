import { screen, render } from '@testing-library/react';

import Button from '../index';

describe('Test Component', () => {
	const renderComponent = () => render(<Button />);

	it('should have label', () => {
		renderComponent();

		expect(screen.getByTestId('button')).toHaveTextContent('Hello from app3');
	});

	it('should match the snapshot', () => {
		const { container } = renderComponent();

		expect(container).toMatchSnapshot();
	});
});
