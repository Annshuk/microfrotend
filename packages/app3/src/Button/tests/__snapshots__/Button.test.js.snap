// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Component should match the snapshot 1`] = `
<div>
  <button
    className="storybook-button storybook-button--medium storybook-button--primary"
    data-testid="button"
    type="button"
  >
    Hello from app3
  </button>
</div>
`;
